config/kubernetes-sigs/org.yaml

```diff
@@ -476,6 +478,7 @@ members:
 - jlamillan
 - jlsong01
 - jmhbnz
+- jmguzik
```

Comment:
need alpha sorting here to fix the failing job

```suggestion
- jmguzik
- jmhbnz
```

Reply:
commit - 4ad04e37e8c23eceeb878f23b4d23d2a92b064c2

Reply:
Fixed, thanks for pointing out this.